modules/microshift-kubeconfig-generating-remote-kcfiles.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_configuring/microshift-kubeconfig.adoc
+
+:_content-type: PROCEDURE
+[id="Generating-additional-kubeconfig-files_{context}"]
+= Generating additional kubeconfig files for remote access
+
+You can generate additional `kubeconfig` files to use if you need more host names or IP addresses than the default remote access file provides.
+
+[IMPORTANT]
+====
+You must restart {product-title} for configuration changes to be implemented.
+====
+
+.Prerequisite
+
+* You have created a `config.yaml` for {product-title}.
+
+.Procedure
+
+. (Optional) You can show the contents of the `config.yaml` by running the following command:
++
+[source, terminal]
+----
+$ cat /etc/microshift/config.yaml
+----
+
+. (Optional) You can show the contents of the remote-access `kubeconfig` file, by running the following command:
++
+[source, terminal]
+----
+$ cat /var/lib/microshift/resources/kubeadmin/<hostname>/kubeconfig
+----
++
+[IMPORTANT]
+====
+Additional remote access `kubeconfig` files must include one of the server names listed in the {product-title} `config.yaml` file. Additional `kubeconfig` files must also use the same CA for validation.
+====
+
+. To generate additional `kubeconfig` files for additional DNS names SANs or external IP addresses, add the entries you need to the `apiServer.subjectAltNames` field. In the following example, the DNS name used is `alt-name-1` and the IP address is `1.2.3.4`.
++
+.Example `config.yaml` with additional authentication values
+[source, YAML]
```

Comment:
Nit: this might not be causing any issues, but `[source, YAML]` should be `[source,yaml]` (applies to all instances)

Reply:
Here, too, I have been told the opposite. I wonder if it doesn't matter?